pkg/printers/humanreadable.go

```diff
@@ -44,6 +40,7 @@ type TablePrinter interface {
 type PrintHandler interface {
 	Handler(columns, columnsWithWide []string, printFunc interface{}) error
```

Comment:
we already switch all `Handler` to `TableHandler` or `DefaultTableHandler` so this function should be also deprecated right ?